doc/sphinxext/cited_mne.py

```diff
@@ -37,12 +44,12 @@
       'Gecko/20100913 Firefox/3.6.9')
 
 # ##### Templates for citations #####
-html = (u""".. _cited
+html = (u""".. _cited:
 
 Publications by users
 =====================
 
-Papers citing MNE as extracted from Google Scholar (on %s).
+Papers citing MNE (%d) as extracted from Google Scholar (on %s).
```

Comment:
Yeah it creates a big diff, but really we shouldn't need to inspect it too closely. I guess we could change to just a bullet list, but it seems like it would be harder to visually keep track of where you are on the page compared to numbers